doc/source/data/doc_code/creating_datasets.py

```diff
@@ -568,13 +568,20 @@
 # GCSFileSystem.
 # NOTE: This example is not runnable as-is; you'll need to point it at your GCS bucket
 # and configure your GCP project and credentials.
-ds = ray.data.read_parquet(
-    "gs://path/to/file.parquet",
-    filesystem=gcsfs.GCSFileSystem(project="my-google-project"),
-)
+path = "gs://path/to/file.parquet"
+filesystem = gcsfs.GCSFileSystem(project="my-google-project")
+ds = ray.data.read_parquet(path, filesystem=filesystem)
 # __read_parquet_gcs_end__
 # fmt: on
 
+
+# fmt: off
+# __validate_parquet_gcs_begin__
+print(filesystem.open(path))
+# <File-like object GCSFileSystem, path/to/file.parquet>
```

Comment:
Another common permission issue is to list directory or file. Shall we also add a print for 

```
print(filesystem.ls(path))
```